src/com/clover/remote/client/CloverConnector.ts

```diff
@@ -1225,6 +1216,14 @@ export namespace CloverConnector {
             this.cloverConnector.broadcaster.notifyOnManualRefundResponse(response);
         }
 
+        public onFinishCancelRefund(result: sdk.remotepay.ResponseCode, reason?: string, message?: string): void {
+            let response: sdk.remotepay.RefundPaymentResponse = new sdk.remotepay.RefundPaymentResponse();
+            CloverConnector.populateBaseResponse(response, false, result,
+                reason ? reason : "Request Canceled",
+                message ? message : "The Manual Refund Request was canceled.");
```

Comment:
This is not a 'manual' refund.  We have two types of refunds - one done with in the context of a payment, and another that is just money back on a card without any payment referenced (naked/manual refund).
Change verbiage here to just "The Refund Request was canceled".

Reply:
Good catch...that's the problem with copying code.  Thanks!